subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/fixture/JUnitMultiVersionIntegrationSpec.groovy

```diff
@@ -38,7 +40,7 @@ import static org.gradle.test.fixtures.junitplatform.JUnitPlatformTestRewriter.*
  */
 abstract class JUnitMultiVersionIntegrationSpec extends MultiVersionIntegrationSpec {
     // JUnit 5's test case name contains parentheses which might break test assertion, e.g. testMethod() PASSED -> testMethod PASSED
-    private static final Pattern TEST_CASE_RESULT_PATTERN = ~/(.*)(\w+)\(\) (PASSED|FAILED|SKIPPED|STANDARD_OUT)/
+    static final Pattern TEST_CASE_RESULT_PATTERN = ~/(.*)(\w+)\(\) (PASSED|FAILED|SKIPPED|STANDARD_OUT)/
```

Comment:
safe?

Reply:
This seems unnecessary - I find no usages outside this class.